PathPlanning/AStar/a_star.py

```diff
@@ -105,10 +105,10 @@ def a_star_planning(sx, sy, gx, gy, ox, oy, reso, rr):
             if tcost >= node.cost:
                 continue  # this is not a better path
 
-            node.cost = tcost
-            openset[n_id] = node  # This path is the best unitl now. record it!
+            node.cost = tcost + calc_heuristic(node, ngoal)
```

Comment:
@tsaoyu Is it correct?

Reply:
This is correct only if `node.cost` refers to *f* cost. I think I am wrong here, cost here is *g* cost.